pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/api/TemplateResource.kt

```diff
@@ -59,6 +59,14 @@ class TemplateResource<Kode : Brevkode<Kode>, out T : BrevTemplate<BrevbakerBrev
             renderHTML(createLetter(kode, letterData, language, felles), letterMarkup)
         }
 
+    fun renderJSON(brevbestilling: BestillBrevRequest<Kode>): LetterMarkup =
```

Comment:
Kanskje alle disse burde være extension functions? Da blir this satt riktig også uten at du trenger with.

Reply:
Liker tanken, men korleis blir det med at desse metodane allereie er inni TemplateResource-klassa?

Reply:
Ah. Nvm